src/otx/algorithms/common/adapters/mmcv/pipelines/load_image_from_otx_dataset.py

```diff
@@ -162,10 +169,13 @@ def _resize_img_ann_if_any(self, results: Dict[str, Any]) -> Dict[str, Any]:
 
     def _load_cache(self, results: Dict[str, Any]) -> Optional[Dict[str, Any]]:
         """Try to load pre-computed results from cache."""
+
         if not self._enable_outer_memcache:
             return None
         key = self._get_unique_key(results)
-        img, meta = self._mem_cache_handler.get(key)
+
+        mem_cache_handler = self._get_memcache_handler()
```

Comment:
It's just a question. It seems that handler is gotten every times this function is called, Is overhead for that neglectable?

Reply:
Yes, no overhead exists.